lib/include/openamp/remoteproc_virtio.h

```diff
@@ -83,6 +83,8 @@ struct remoteproc_virtio {
 	struct virtio_device vdev;
 };
 
+extern virtio_dispatch remoteproc_virtio_dispatch_funcs;
```

Comment:
alternative would be to initialized it in rproc_virtio_create_vdev  func or too late? 
	vdev->reset_cb = rst_cb;
	vdev->vrings_num = num_vrings;
  +      vdev->func = remoteproc_virtio_dispatch_funcs;